pkg/util/cluster_info.go

```diff
@@ -1,6 +1,15 @@
 package util
 
-import "sync"
+import (
+	"flag"
+	"sync"
+	"time"
+)
+
+var (
+	// Make the vSphere call timeout configurable.
+	Timeout = flag.Duration("vmware-timeout", 5*time.Minute, "Timeout of all VMware calls")
```

Comment:
I had to move this here for breaking some cycling dependency, so as it can be used from both cache package and check package. If we keep it in either of them, then it causes a cycle.. :(